### PR TITLE
Improve and simplify the contributing instructions about tests

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: contributing/code/tests.rst [hunk 1]
```diff
@@ -3,54 +3,32 @@
 Running Symfony Tests
 =====================
 
-Before submitting a :doc:`patch <patches>` for inclusion, you need to run the
-Symfony test suite to check that you have not broken anything.
+The Symfony project uses a third-party service which automatically runs tests
+for any submitted :doc:`patch <patches>`. If the new code breaks any test,
+you'll be notified in detail about that error.
```
Comment (stof) [inline contributing/code/tests.rst]: you should mention that this will be in the PR status. The contributor will not be notified by email

Comment (javiereguiluz) [inline contributing/code/tests.rst]: I agree with you. That phrase has been reworded. Thanks!

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Pull request closed without merging**